.travis.yml

```diff
@@ -1,11 +1,12 @@
 language: go
 go:
-- 1.7
-- 1.8
-- 1.9
-- tip
+- "1.7"
+- "1.8"
+- "1.9"
+- "1.10"
+- "master"
```

Comment:
`tip` is a legacy reference from when the Go project used mercurial instead of git.